packages/storybook/utils/FilledBox.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function FilledBox({ width, height, color, children }) {
+    const style = {
+        background: color,
+        width,
+        height,
+    };
+    return (
+        <div style={style}>
+            {children}
+        </div>
+    );
+}
+
+FilledBox.propTypes = {
+    width: PropTypes.oneOfType([
+        PropTypes.number,
+        PropTypes.string
+    ]),
+    height: PropTypes.oneOfType([
+        PropTypes.number,
+        PropTypes.string
+    ]),
+    color: PropTypes.string,
+};
+
+FilledBox.defaultProps = {
+    width: undefined,
+    height: undefined,
```

Comment:
From the name `FilledBox`, I thought that it might be a full width and full height box by default. Maybe the name could be changed as `ColoredBox`?

Reply:
Sounds good to me 👍